packages/third-parties/pulse/src/services/PulseService.ts

```
@@ -0,0 +1,158 @@
+          pulse,
+          name,
+          (job: Job) => {
+            const $ctx = new DIContext({
+              id: uuid()
+            });
+
+            $ctx.set("job", job);
+
+            return runInContext($ctx, () => processor(job));
+          },
+          options);
```

ESLint

[error] Insert `⏎········`

(prettier/prettier)